### PR TITLE
Enhancement: Lint clean-up basic PEP8 standards.

### PR DESCRIPTION
**Pull request by susieoneil**
## About The Pull Request

Hi, this is my first contribution to ClanGen.  I wanted to start small just to get a feel for the pull request process before I tackle more.  This request aims to clean up the code to some basic PEP8 standards.  I have started with the 'tests' folder first as those files mostly just contain whitespace fixes and a few local scope variable names that could be easily changed. I would like to continue with the scripts folder if you are happy with this one. I left anything I wasn't quite sure of yet, especially if it looked like a work in progress.
- Added, removed or fixed some blank lines, spaces/tabs, indents, long lines.
- Moved top level imports to top and removed some unused/duplicate ones.
- Renamed a few local variables.
- Fixed some spelling/grammar in comments. (Changing 'dont' to 'do not' was just to avoid adding apostrophes. Either would be fine though).

## Why This Is Good For ClanGen

This improves the readability of the code which makes it easier to work on other issues or find new ones.  Removing the simple lint should make the Pylint report more useful for seeing the more critical errors and probably run a bit faster.

I asked a few weeks ago on the Discord coding-and-modding channel, if PEP8 fixes would be welcome.  I got an enthusiastic reply from Lily and a like from Fable, so here it is  :)
https://discord.com/channels/1003759225522110524/1003759339359699054/1235663167909269627

## Proof of Testing

- I started the game via the run.bat file with no issues.
- I made 2 batch files to run all the existing tests locally. tests.bat runs all of them. tests_cover.bat runs most of them with a single output. I've included a snippet from the coverage output, showing the tests ran ok.
image
image
image

### CODE REVIEW

File: tests/test_save.py [hunk 1]
```diff
@@ -107,7 +108,7 @@ def example_save(self, id):
         if os.path.exists(get_save_dir()):
             shutil.rmtree(get_save_dir())
 
-        #copy tests/testSaves/save<id> to saves
+        # tests/testSaves/save<id> to saves
```
Comment (Thlumyn) [inline tests/test_save.py]: Why are we removing the word copy here?

**Comment by Thlumyn** [comment]
It looks good otherwise, imo, it's nice to see some file cleanups.

**Comment by scribblecrumb** [comment]
Just wanted to comment that if you do move onto working on the Scripts, avoid the events_module and events.py for a little bit.  I'm doing quite a lot of refactoring work in there right now and wouldn't want you to waste your time on something that might have moved or changed completely soon haha

**Review by Tiritirimatangi — COMMENTED** [review]
I'm not qualified to review coding changes of this complexity (or, err, almost any complexity lol) but can I just say thank you so much for refactoring things here <3

File: tests/test_save.py (hunk 1 above)
Comment (susieoneil) [reply tests/test_save.py]: Oh sorry, that looks like a mistake.  Probably accidentally selected the word while trying to add a space.

Comment (susieoneil) [reply tests/test_save.py]: Fixed it now.

**Comment by susieoneil** [comment]
@scribblecrumb 
Thanks for that, I'll leave those alone for now.  I might just do one subfolder at a time, to make the commits easy to keep tidy.

**Review by Thlumyn — APPROVED** [review]

**Pull request merged**